apps/playground/src/components/playground/chat-page-client.tsx

```
@@ -753,15 +753,15 @@ export default function ChatPageClient({
-			for (const submit of submitFns) {
-				try {
-					await submit(content);
-				} catch (mirrorError) {
+			const results = await Promise.allSettled(
+				submitFns.map((submit) => submit(content)),
+			);
+			for (const result of results) {
+				if (result.status === "rejected") {
 					// Don't surface comparison errors as hard failures
-
 					console.warn(
```

ESLint

[error] Unexpected console statement.

(no-console)